Tools/WinMLDashboard/public/requirements.txt

```diff
@@ -1,6 +1,5 @@
 git+https://github.com/apple/coremltools
---extra-index-url http://aipkgs/repository/aiinfra/simple --trusted-host aipkgs 
-libsvm==3.22
+lightgbm==2.2.1
 winmltools==1.2.0.912
 tensorflow==1.11.0
```

Comment:
Why is tensorflow needed?

Reply:
Tensorflow is needed for Keras. Keras can use tensorflow as backend.